### ISSUE TITLE
Pytorch backend problem

### ISSUE THREAD

**Issue by xiaolling** [BUTSpeechFIT/VBx#36]
Hi, I try to use the pytorch backend. I used the ResNet101_16kHz/nnet/raw_81.pth.zip* file (model file md5sum: 4dbae96de5392a9db41193b1aebdfe24 ), but I got error and the log : "Unexpected key(s) in state_dict: "metric.weight". I'm not sure if I'm using it incorrectly. Does it support the pytorch backend now? Thanks......
